pkg/forwarder/domain_forwarder.go

```diff
@@ -144,11 +147,30 @@ func (f *domainForwarder) handleFailedTransactions() {
 	}
 }
 
+// scheduleConnectionResets signals the workers to recreate their connections to DD
+// at the configured interval
+func (f *domainForwarder) scheduleConnectionResets() {
+	ticker := time.NewTicker(f.connectionResetInterval)
```

Comment:
It's a nit but I think we should have a minimum here so customer don't shoot themselves in the foot (like setting the value to `1` thinking it's hours). Maybe just a minimal value, 1min maybe, and an error message should be enough.

Reply:
yeah, I've considered this as well, but overall it's not super dangerous to set this to the minimum value of `1 second` by mistake: it'll make the ticker tick every second, and make the agent create a new connection for almost every transaction.

I'll leave this as-is for now, but let's reconsider if/when the option becomes public and documented though